app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(commands_str: str) -> None:
+    commands_list, file_out, file_in = commands_str.split()
+    if (len(commands_list) == 3 and commands_list[0] == "cp"
```

Comment:
This won't work. The variable `commands_list `will be a `str `and equal to `cp`, so the condition will be false.